frontend/src/components/icons/ArtistIcon.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from 'react'
+import ArtistLogo from '../../assets/icons/icons8-micro-50.png'
+
+export default class ArtistIcon extends Component {
+    render() {
+        return (
+            <img src={ArtistLogo} alt="" width="32px" height="32px" className="icon"/>
+        )
+    }
+}
```

Comment:
Delete these component SVG files

Reply:
Need to convert ArtistIcon and TrackIcon to SVGs.